nrf_modem/doc/gnss_interface.rst

```diff
@@ -0,0 +1,534 @@
+interval is short.
+
+The default value is NRF_MODEM_GNSS_TIMING_SOURCE_RTC.
+
+.. note::
+
+   Use of TCXO significantly raises the idle current consumption.
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_timing_source_set(NRF_MODEM_GNSS_TIMING_SOURCE_TCXO);
+
+.. note::
+
+   This feature is only supported by modem firmware v1.3.0 or later.
+
+QZSS configuration
+==================
+
+GNSS has configuration options which can be used to change QZSS related behavior.
+
+.. note::
+
+   QZSS is only supported by modem firmware v1.3.0 or later.
+
+NMEA mode
+---------
+
+QZSS NMEA mode controls whether QZSS satellites are reported in NMEA strings or not. NMEA 4.10 standard does not support QZSS
+satellites, so in standard NMEA mode QZSS satellites are not reported in GPGSA and GPGSV sentences. In custom NMEA mode satellite
+IDs 193...202 are used for QZSS satellites.
+
+The default value is NRF_MODEM_GNSS_QZSS_NMEA_MODE_STANDARD.
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_qzss_nmea_mode_set(NRF_MODEM_GNSS_QZSS_NMEA_MODE_CUSTOM);
+
+.. note::
+
+   QZSS is only supported by modem firmware v1.3.0 or later.
+
+PRN mask
+--------
+
+QZSS satellite acquisition and tracking can be configured per satellite using QZSS PRN mask. Bits 0...9 correspond to QZSS PRNs
+193...202, respectively. When a bit is set, usage of the corresponding QZSS satellite is enabled. Bits 10...15 are reserved and
+their value is ignored.
+
+The default is that all QZSS PRNs (193...202) are enabled.
+
+QZSS PRNs 193, 194, 195 and 199 can be enabled (and others disabled) as shown in the following example:
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_qzss_prn_mask_set(0x47);
+
+.. note::
+
+   QZSS is only supported by modem firmware v1.3.0 or later.
+
+Enabling GNSS priority mode
+***************************
+
+GNSS can be given priority over LTE idle mode procedures to help getting a fix. Usually this is not necessary when eDRX and/or
+PSM is used, but if that's not possible, the GNSS priority mode may be used.
+
+Priority for GNSS should be used only when fix has been blocked by LTE idle mode operations, which can be detected by
+NRF_MODEM_GNSS_PVT_FLAG_NOT_ENOUGH_WINDOW_TIME bit being set in the PVT data frame flags member. The application should not
+make the decision based on a single PVT event, but should enable priority only in case this flag has been set in several
+consecutive PVT events.
+
+Priority mode is disabled automatically after producing the first fix. It can also be disabled by the application by calling
+:c:func:`nrf_modem_gnss_prio_mode_disable` function.
+
+.. note::
+
+   GNSS priority may interfere with LTE operations, if possible it would be good to time the use of priority to moments where
+   data transfer is not anticipated. In general, long enough eDRX cycles, or PSM, will ensure better functionality for both
+   GNSS and LTE.
```

Comment:
```suggestion
   GNSS priority may interfere with LTE operations.
   If possible, it would be good to time the use of priority to moments where data transfer is not anticipated.
   In general, eDRX cycles that are long enough, or PSM, ensure better functionality for both GNSS and LTE.
```